tests/testthat/test-derive_param_merged_exist_flag.R

```diff
@@ -0,0 +1,80 @@
+adsl <- tibble::tribble(
+  ~USUBJID,
+  "1",
+  "2",
+  "3"
+) %>%
+  mutate(STUDYID = "XX1234")
+
+adrs <- tibble::tribble(
+  ~USUBJID, ~AVALC, ~PARAMCD,
+  "1",      "PR",   "OVR",
+  "1",      "CR",   "OVR",
+  "1",      "CR",   "OVR",
+  "1",      "SD",   "OVR",
+  "2",      "SD",   "OVR",
+  "2",      "PD",   "OVR",
+  "2",      "PD",   "OVR",
+  "1",      "PR",   "OVRF",
+  "1",      "CR",   "OVRF",
+  "1",      "CR",   "OVRF",
+  "1",      "PD",   "OVRF",
+  "2",      "SD",   "OVRF",
+  "2",      "PD",   "OVRF",
+  "2",      "PD",   "OVRF"
+) %>%
+  mutate(
+    STUDYID = "XX1234",
+    ANL01FL = "Y"
+  )
+
+# derive_param_merged_exist_flag ----
+## derive_param_merge_exist_flag Test 1: derive parameter indicating PD ----
```

Comment:
Maybe this is ok for testing: but once again i think the PD param needs a date together with AVALC